### ISSUE TITLE
Parsing error of `'^(\".*\"|\''.*\'')$'`

### ISSUE THREAD

**Issue by steve02081504** [Tatakinov/yayalint#15]
e.g.
```c
IsStringToken{
	RE_GREP(_argv[0],'^(\".*\"|\''.*\'')$')
}
```
![图片](https://github.com/Tatakinov/yayalint/assets/31927825/04608d48-651c-405e-9827-094f19476cfb)

**Comment by steve02081504** [comment]
another error:
```c
TalkSakuraHeadPat.0{
	if !RAND(MAX(7,-intimacy-7))
		intimacy++
}
```

**Comment by Tatakinov** [comment]
maybe fixed in v1.8.4

**Comment by steve02081504** [comment]
fixed!
thanks for your work
XD
